tests/VersionRouteTest.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+namespace RonasIT\Support\Tests;
+
+use Illuminate\Support\Facades\Route;
+use RonasIT\Support\Contracts\VersionEnumContract;
+use RonasIT\Support\Enum\VersionEnum;
+use RonasIT\Support\Routing\RouteFacade;
+
+class VersionRouteTest extends HelpersTestCase
+{
+    protected const ROUTE_FACADE = '/test-facade';
+    protected const ROUTE_OBJECT = '/test-object';
+
+    protected bool $init = false;
```

Comment:
```suggestion
    protected static bool $init = false;
```

Reply:
It turned out that this field is not needed at all. Because routes are registered in the test, they are deleted after the test completes.